content/700-optimize/200-getting-started.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Getting Started'
+metaTitle: 'Getting started with Prisma Optimize'
+metaDescription: 'Learn how to quickly set up and start using Prisma Optimize.'
+tocDepth: 3
+toc: true
+---
+
+## Prerequisites
+
+Before you begin with Prisma Optimize, ensure you have the following:
+
+- A [Prisma Data Platform account](https://console.prisma.io).
+- A project using [Prisma Client](/orm/prisma-client) version `5.0.0` or higher (we recommend using the latest version).
+- A PostgreSQL, MySQL/MariaDB, PlanetScale, CockroachDB, MS SQL Server, or MongoDB database.
+
+## 1. Launch Optimize
+
+1. Log in to your [Prisma Data Platform account](https://console.prisma.io/optimize).
+2. Access and launch the Optimize dashboard by following the instructions [here](/platform/about#accessing-the-optimize-dashboard).
+
+## 2. Add Optimize to your application
+
+### 2.1. Update Your `schema.prisma` file
+
+In the `generator` block of your Prisma schema, add the `tracing` preview feature:
+
+```prisma
+generator client {
+  provider = "prisma-client-js"
+  previewFeatures = ["tracing"]
+}
+```
+
+Then, generate the Prisma Client:
+
+```bash
+npx prisma generate
+```
+
+### 2.2. Install the Optimize Prisma Client extension
+
+Install the latest versions of Prisma Client and the Optimize extension:
+
+```bash
+npm install @prisma/client@latest @prisma/extension-optimize
+```
+
+### 2.3. Add the Optimize API Key to your `.env` file
+
+Generate an Optimize API key by following the instructions [here](/platform/about#generating-an-optimize-api-key) and add it to your `.env` file:
```

Comment:
open in new tab

Reply:
@jharrell could you help me on this? I'm not sure how I'd do that, as the following syntax is seemingly not working `[here](/platform/about#generating-an-optimize-api-key){:target="_blank"}`.

Reply:
```suggestion
<a href="https://www.prisma.io/docs/platform/about#generating-an-optimize-api-key" target="_blank">Generate a Prisma Optimize API key</a> and add it to your `.env` file:
```